itest/itest.sh

```diff
@@ -73,7 +74,13 @@ function main {
   if [[ "$1" = "docker" ]]; then
     echo "Building itest image"
     # Build the base image
-    docker build -t pidtree-itest-base .
+    if [ -f /etc/lsb-release ]; then
+      source /etc/lsb-release
+    else
+      echo "WARNING: Could not source /etc/lsb-release, tentatively creating bionic docker image"
+      DISTRIB_CODENAME=bionic
+    fi
```

Comment:
why is this block necessary?

Reply:
Because itests need to install kernel headers, and if the docker container is not on the same distro there is a good chance that you will need to forward port the headers packages. The problem does not exist if you run a Docker image with the same ubuntu release.